Parser/asdl_c.py

```diff
@@ -998,17 +998,20 @@ def visitModule(self, mod):
         self.emit("if (!init_types()) return NULL;", 1)
         self.emit('m = PyState_FindModule(&_astmodule);', 1)
         self.emit("if (!m) return NULL;", 1)
+        self.emit('if (PyModule_AddObject(m, "AST", astmodulestate_global->AST_type) < 0) goto error;', 1)
```

Comment:
Please add braces, see PEP 7.

```suggestion
        self.emit('if (PyModule_AddObject(m, "AST", astmodulestate_global->AST_type) < 0) { goto error; }', 1)
```

Same for changes below.